Simplenote/CoreDataManager.swift

```diff
@@ -0,0 +1,102 @@
+import Foundation
+import CoreData
+
+@objcMembers
+class CoreDataManager: NSObject {
+
+    /// URL for the managed object model resource
+    ///
+    static private let modelURL: URL? = {
+        guard let path = Bundle.main.path(forResource: Constants.resourceName, ofType: Constants.resourceType) else {
+            return nil
+        }
+        return URL(string: path)
+    }()
+
+    /// URL for the in app documents directory
+    ///
+    static let documentsDirectory: URL = {
+        FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first!
+    }()
+
+    /// In app core data storge URL
+    ///
+    static let appStorageURL: URL = {
+        documentsDirectory.appendingPathComponent(Constants.sqlFile)
+    }()
+
+    /// URL for Simplenote's shared app group directory
+    ///
+    static let groupDirectory: URL = {
+        FileManager.default.containerURL(forSecurityApplicationGroupIdentifier: Constants.sharedDirectoryDomain + Constants.groupIdentifier)!
+    }()
+
+    /// URL for Simplenote's shared app group documents directory
+    ///
+    static let groupDocumentsDirectory: URL = {
+        groupDirectory.appendingPathComponent(Constants.documentDirectory)
+    }()
```

Comment:
The following properties aren't really CoreData-Y

- documentsDirectory
- appStorageURL
- groupDirectory
- groupDocumentsDirectory

WDYT about relocating those to `FileManager+Simplenote` or so? (they should probably not be `static let` but `var` instead, accessible thru the FileManager instance.